src/main/java/mx/edu/utez/pret/controller/IdiomaCandidatoController.java

```diff
@@ -108,16 +108,14 @@ public ResponseEntity<Map<String, Object>> borrar(@RequestHeader HttpHeaders hea
             for (IdiomaCandidato idiomaCandidato : candidato.getIdiomas()) {
```

Comment:
La idea de crear el QueryMethod, es para eliminar este for y evitar consultar todos los idiomas del candidato, la idea del funcionamiento es la siguiente:
- Consulto si existe el registro específico mediante el id del candidato y el id del idioma en la tabla
- Si existe, eliminamos y mandamos mensaje de éxito.
- Si no existe, solo avisamos que no eliminamos nada.